python/tvm/script/builder/tir/prim_func_frame.py

```diff
@@ -26,6 +26,7 @@
 
 from . import _ffi_api
 from .base import TIRFrame
+from ...parse.utils import extra_vars
```

Comment:
let's move this into `prim_func` to avoid cyclic dependency - the parser depends on the builder heavily already, so we cannot let the builder depend on the parser